bazelisk.py

```diff
@@ -34,6 +34,8 @@
 
 ONE_HOUR = 1 * 60 * 60
 
+OPERATING_SYSTEM = platform.system().lower()
```

Comment:
I see why you added this, but I think when the second comment is addressed, we can remove this cached global again and just use the local variable as before.

Reply:
Indeed! So I'm removing it locally and I will push as soon as I am ready. :)

Reply:
Done.